Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFloor.cs

```diff
@@ -240,17 +240,16 @@ private ICurve GetFlattenedCurve(ICurve curve, double z)
 
         case OG.Polyline poly:
           var polylinePonts = new List<double>();
-          var originalPolylinePoints = poly.ToList();
+          var originalPolylinePoints = poly.points;
```

Comment:
The compiler warns that Polyline.points is obsolete and we should use Polyline.GetPoints(). Not sure if this actually matters though because they both do the same thing